core/src/main/scala/kanaloa/reactive/dispatcher/Dispatcher.scala

```diff
@@ -144,7 +146,8 @@ object PushingDispatcher {
     backend:    T,
     rootConfig: Config = ConfigFactory.load()
   )(resultChecker: ResultChecker)(implicit system: ActorSystem) = {
-    val (settings, metricsCollector) = Dispatcher.readConfig(name, rootConfig)
+    val (settings, reporter) = Dispatcher.readConfig(name, rootConfig)
```

Comment:
it's essential to kanaloa, without which there would be no back pressure, auto scaling or realtime monitoring, maybe except circuit breaker. I am not sure if anyone would want to use kanaloa just for circuit breaker.